UnicornRender/video/source/OrthographicCamera.cpp

```diff
@@ -36,6 +36,7 @@ void OrthographicCamera::UpdateProjection()
     m_cameraProjection = glm::ortho(-xScaled, xScaled,
                                     -yScaled, yScaled,
                                     m_back, m_front);
+    m_cameraProjection[1][1] *= -1; //ToDo: For vulkan rendering.
```

Comment:
what is this `todo` about?

Reply:
This was moved from shader here.We inverting perspective in vertical axis to render it correctly, because vulkan has it upside-down

Reply:
I think that it's not a todo than, but a comment.